src/GraphQL/schema/products.graphql

```diff
@@ -33,6 +33,7 @@ type Product {
     dimensions: ProductDimensions
     weight: ProductWeight
     images: [Image!]
+    tags: [Tag!]! @hasMany
```

Comment:
@AlexeiaW I think right here the last ! means that tags are required for product, which won't be the case. We do need to make sure that the array only accepts tag which is correct as you have it, but not that a minimum amount of tags is required in the product

Reply:
Ok, I made the tag optional